howdoi/howdoi.py

```diff
@@ -487,6 +508,14 @@ def command_line_runner():
             _print_err('Clearing cache failed')
         return
 
+    if args['stash_view']:
+        os.system('keep list')
```

Comment:
Could we add some basic error handling here? Looks like `os.system` returns the exit status, but could be a bit more complex on other operating systems. https://docs.python.org/3/library/os.html#os.system